src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,10 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public static Color getRandomColor() {
```

Comment:
Don't use static methods in your solution
Static methods are in general a bad practice. Let's better create an instance of a class which method you want to call.